qemu/tests/multi_disk_random_hotplug.py

```diff
@@ -179,7 +179,6 @@ def insert_into_qdev(qdev, param_matrix, no_disks, params, new_devices):
                 for dev in devs:
                     if dev in qdev:
                         qdev.remove(dev, recursive=True)
-                formats.remove(fmt)
```

Comment:
I have updated the code, now I add codes that only won't to remove the 'fmt' for qid existed issue in function insert_into_qdev()